<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-B992BI1B" failures="3" tests="6" name="Test1" time="58.198" errors="0" timestamp="2022-07-14T16:18:28 IST">
  <testcase classname="com.nopCommerce.testcases.TC_LoginTest_001" name="loginTest" time="12.671"/>
  <testcase classname="com.nopCommerce.testcases.TC_LoginDDT_002" name="loginTest" time="10.815"/>
  <testcase classname="com.nopCommerce.testcases.TC_LoginDDT_002" name="loginTest" time="4.037">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.nopCommerce.testcases.TC_LoginDDT_002.loginTest(TC_LoginDDT_002.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="com.nopCommerce.testcases.TC_LoginDDT_002" name="loginTest" time="3.87">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.nopCommerce.testcases.TC_LoginDDT_002.loginTest(TC_LoginDDT_002.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="com.nopCommerce.testcases.TC_LoginDDT_002" name="loginTest" time="4.221">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.nopCommerce.testcases.TC_LoginDDT_002.loginTest(TC_LoginDDT_002.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="com.nopCommerce.testcases.TC_Addcustomertest_003" name="addNewCustomer" time="13.797"/>
</testsuite> <!-- Test1 -->
